topology/src/main/java/io/camunda/zeebe/topology/api/TopologyManagementRequestSender.java

```diff
@@ -111,7 +110,7 @@ public CompletableFuture<Either<ErrorResponse, TopologyChangeResponse>> forceSca
         forceScaleDownRequest,
         serializer::encodeScaleRequest,
         serializer::decodeTopologyChangeResponse,
-        coordinator,
+        coordinatorSupplier.getNextCoordinator(forceScaleDownRequest.members()),
```

Comment:
`TopologyCordinatorSupplier` is aware of the current topology and choses either broker 0 or other one based on the current topology.  We cannot use a single method, because for force scale down it should look at the remaining brokers that will be available after the request is applied and for other requests it should use the current brokers.

Reply:
Right but what I meant is this:
1. One region goes down.
2. We decide force scale down and only keep the surviving region
4. This works because a ForceScaleDownRequest is not always sent to broker 0 but the lowest remaining broker.
5. Eventually we want to recover from this.
6. We should be able to scale up, adding temporary brokers or the recovered brokers from the other region.
7. We can't scale up because all scale-up requests are sent to broker 0 which isn't active _yet_

Reply:
> We can't scale up because all scale-up requests are sent to broker 0 which isn't active yet

This is not true. At this point `TopologyCoordinatorSupplier` uses the current topology to find the coordinator which will be the next available lowest brokerId.

Reply:
Oh :facepalm: Thanks for the explanation :+1: